onnxruntime/contrib_ops/cpu/quantization/qlinear_util.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef ONNXRUNTIME_QLINEAR_UTIL_H
+#define ONNXRUNTIME_QLINEAR_UTIL_H
+#include <cstdint>
+#include "core/framework/tensor.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+constexpr int LOOKUP_TABLE_IS_FIXED = 1;
+constexpr int LOOKUP_TABLE_IS_COPY = 2;
+
+static inline bool has_same_scale(const Tensor* tensor_x_scale, const Tensor* tensor_y_scale) {
```

Comment:
It is duplicate with functions in qlinear_concat.cc. Please remove the duplication.